tests/cli.rs

```diff
@@ -530,6 +535,66 @@ fn user_password_auth() {
     mock.assert();
 }
 
+#[test]
+fn netrc_env_user_password_auth() {
+    let server = MockServer::start();
+    let mock = server.mock(|when, _then| {
+        when.header("Authorization", "Basic dXNlcjpwYXNz");
+    });
+
+    let mut netrc = tempfile::NamedTempFile::new().unwrap();
+    writeln!(
+        netrc,
+        "machine {}\nlogin user\npassword pass",
+        server.host()
+    )
+    .unwrap();
+
+    get_command()
+        .env("NETRC", netrc.path())
+        .arg(server.base_url())
+        .assert();
+    mock.assert();
+}
+
+#[test]
+fn netrc_file_user_password_auth() {
+    for netrc_file in [".netrc", "_netrc"].iter() {
+        let server = MockServer::start();
+        let mock = server.mock(|when, _then| {
+            when.header("Authorization", "Basic dXNlcjpwYXNz");
+        });
+
+        let homedir = tempfile::TempDir::new().unwrap();
+        let netrc_path = homedir.path().join(netrc_file);
+        let mut netrc = File::create(&netrc_path).unwrap();
+        writeln!(
+            netrc,
+            "machine {}\nlogin user\npassword pass",
+            server.host()
+        )
+        .unwrap();
+
+        netrc.flush().unwrap();
+
+        #[cfg(target_os = "win")]
+        get_command()
+            .env("XH_TEST_MODE_WIN_HOME_DIR", homedir.path())
+            .arg(server.base_url())
+            .assert();
+
+        #[cfg(not(target_os = "win"))]
+        get_command()
+            .env("HOME", homedir.path())
+            .arg(server.base_url())
+            .assert();
```

Comment:
Not sure if it'll help, but it should be safe to roll these into one. `HOME` won't hurt Unix tests and `XH_TEST_MODE_WIN_HOME_DIR` won't hurt Windows tests.